controllers/idler/idler_controller.go

```diff
@@ -35,7 +35,7 @@ import (
 
 const (
 	RestartThreshold     = 50
-	RequeueTimeThreshold = 300 * time.Second
+	RequeueTimeThreshold = 6 * time.Hour
```

Comment:
I wonder if we can make it a bit shorter like 3h? Basically crash-lopping pods will be killed after the threshold is reached (50 restarts) plus up to `RequeueTimeThreshold`. With this change it's up to 6h instead of 5m extra time after the threshold. Which is fine. But potentially could be shorter (3h???) if it's not too often for the idler with many namespaces.
But again, it's minor. We can keep it as 6h for now and shorten it after we switch to a pod watcher.

Reply:
+1